### PR TITLE
Use multiline literal style for long_description string.

### PR DESCRIPTION
**Pull request by samgavinio**
/cc @zendesk/mintegrations @zendesk/i18n @dadah89

### Description

Use multiline literal style for long_description string.

Addresses discussion from: https://github.com/zendesk/shopify_app/pull/76#issuecomment-311827079

### Risks
* [low] Minor english string edits

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
...
       "title": "Installation Instructions for App: Shopify",
-      "value": "1. After installation, click the setup button that appears inside the Shopify app. n2. You will be redirected to the Zendesk app in Shopify marketplace.\n3. Click get to install the Zendesk app.\n4. Open the Zendesk app in Shopify and click \"\"I have an account\"\".\n5. Enter your Zendesk Support subdomain and click next\n6. Click allow to provide the integration access privilege to your Zendesk Support account.\n7. The integration will automatically setup the Shopify app in Zendesk Support and web widget inside your Shopify storefront."
+      "value": "1. After installation, click the setup button that appears inside the Shopify app.\n2. You will be redirected to the Zendesk app in the Shopify marketplace.\n3. Click get to install the Zendesk app.\n4. Open the Zendesk app in Shopify and click \"I have an account\".\n5. Enter your Zendesk Support subdomain and click next\n6. Click allow to provide the integration access privilege to your Zendesk Support account.\n7. The integration will automatically setup the Shopify app in Zendesk Support and web widget inside your Shopify storefront."
     },
```
Comment (mariasica) [inline src/translations/en.json]: web widget is a product name and should be capitalized

File: src/translations/en.yml [hunk 2]
```diff
@@ -16,33 +16,29 @@ parts:
...
+        The Zendesk Support for Shopify app works to unite your business by displaying critical Shopify data beside Zendesk Support ticket information.
 
+        The application queries your Shopify store to find the customer’s details and recent orders. In addition, it displays order summary information if a ticket is linked to an individual order.
 
-The application queries your Shopify store to find the customer’s details and recent orders. In addition, it displays order summary information if a ticket is linked to an individual order.
+        ### Improve customer satisfaction with better conversations
 
+        * Solve tickets faster by having access to relevant customer data from Shopify, such as billing, shipping, and order information, on Zendesk Support ticket pages.
+        * Easily navigate to additional data in Shopify by simply clicking the Order ID.
+        * Add a Zendesk Feedback Tab to any Shopify-powered site so your customers can contact your Support team quickly and easily.
```
Comment (mariasica) [inline src/translations/en.yml]: the feedback tab does not exist anymore

**Comment by samgavinio** [comment]
Hi @mariasica 💇 , please have a look again now. Thanks.

**Comment by dadah89** [comment]
@samgavinio actually told you guys the wrong information, the issue before happened with incorrectly escaped quotes (fixed here https://github.com/zendesk/shopify_app/pull/78). You can just keep the fixed quotes or use that style of multiline, whatever you think it's best

File: src/translations/en.yml [hunk 3]
```diff
@@ -16,33 +16,29 @@ parts:
...
-3. Click get to install the Zendesk app.
-4. Open the Zendesk app in Shopify and click \"I have an account\".
-5. Enter your Zendesk Support subdomain and click next
-6. Click allow to provide the integration access privilege to your Zendesk Support account.
-7. The integration will automatically setup the Shopify app in Zendesk Support and web widget inside your Shopify storefront."
+        value: |-
+          1. After installation, click the setup button that appears inside the Shopify app.
+          2. You will be redirected to the Zendesk app in the Shopify marketplace.
+          3. Click get to install the Zendesk app.
+          4. Open the Zendesk app in Shopify and click \"I have an account\".
```
Comment (dadah89) [inline src/translations/en.yml]: using multiline, you don't need to escape quotes here

**Comment by samgavinio** [comment]
@dadah89 Thanks. We've decided to keep the multiline here since it is makes more for a more readable YML. Plus this also addresses the string issues brought up by Maria.

**Comment by dadah89** [comment]
@samgavinio cool. I checked that the file pass the i18n validations (and seems to compile fine as yaml). Should be good to merge after you get +1 from the Maria/Olena

**Comment by mariasica** [comment]
Please let;s stick to normal lines with escaped quotes next time. The only other team who uses this is BIME. Better to stick to what is more common to keep consistency which will avoid issues when doc team or other team members edit strings. This is ok 👍

**Pull request merged**